lib/ts/Diags.cc

```diff
@@ -816,29 +820,39 @@ Diags::set_stdout_output(const char *_bind_stdout)
   if (strcmp(_bind_stdout, "") == 0)
     return false;
 
-  if (stdout_log) {
-    delete stdout_log;
-    stdout_log = NULL;
-  }
-
-  // create backing BaseLogFile for stdout
-  stdout_log = new BaseLogFile(_bind_stdout);
+  BaseLogFile *old_stdout_log = stdout_log;
+  BaseLogFile *new_stdout_log = new BaseLogFile(_bind_stdout);
```

Comment:
Can we find a better argument name than ``_bind_stdout``? Maybe ``path`` or ``stdout_path``?